src/OctoshiftCLI.Tests/BbsApiTests.cs

```diff
@@ -0,0 +1,97 @@
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using OctoshiftCLI.Extensions;
+using Xunit;
+
+namespace OctoshiftCLI.Tests
```

Comment:
nit: CodeQL now understands `file-scoped namespaces`, so we should use it for new files and also refactor the old ones as we touch them. It will reduce the indentation.